DeceptionGame/Assets/Scripts/PlayerController.cs

```diff
@@ -13,6 +14,14 @@ public class PlayerController : MonoBehaviour {
     [SerializeField] private float _interactionRadius = 0;
     private Collider[] _nearbyInteractables = null;
 
+    public void Initialize(bool mainPlayer) {
```

Comment:
I expect this to be changed around when networking and the concept of the "main player" gets more defined